share/spack/gitlab/cloud_pipelines/stacks/radiuss/spack.yaml

```diff
@@ -0,0 +1,128 @@
+        java: [openjdk, jdk, ibm-java]
+        jpeg: [libjpeg-turbo, libjpeg]
+        lapack: [openblas, amdlibflame]
+        mariadb-client: [mariadb-c-client, mariadb]
+        mkl: [intel-mkl]
+        mpe: [mpe2]
+        #mpi: [openmpi, mpich]
+        mysql-client: [mysql, mariadb-c-client]
+        opencl: [pocl]
+        osmesa: [mesa+osmesa, mesa18+osmesa]
+        pil: [py-pillow]
+        pkgconfig: [pkgconf, pkg-config]
+        rpc: [libtirpc]
+        scalapack: [netlib-scalapack, amdscalapack]
+        sycl: [hipsycl]
+        szip: [libszip, libaec]
+        tbb: [intel-tbb]
+        unwind: [libunwind]
+        yacc: [bison, byacc]
+        uuid: [util-linux-uuid, libuuid]
+  definitions:
+    #- compilers: ['%gcc@8.3.1', '%clang@10.0.0']
+    - compilers: ['%gcc@8.3.1']
+    
+    # Note skipping spot since no spack package for it
+    - radiuss:
+      - ascent  # ^conduit@0.6.0
+      - axom
+      - blt
+      - caliper
+      - care
+      - chai
+      - conduit  # ^hdf5+shared
+      - flux-core
+      - flux-sched
+      - glvis   # ^mesa-glu@9.0.0 ^mesa18~llvm
+      - hypre
+      - lbann
+      - lvarray # ^raja~openmp  # per Ben Corbett Issue #23192
+      - mfem
+      - py-hatchet
+      - py-maestrowf
+      - py-merlin
+      - py-shroud
+      - raja
+      - samrai
+      - scr
+      - sundials
+      - umpire ~openmp
+      - visit   # ^mesa-glu@9.0.0
+      - xbraid
+      - zfp
+
+  mirrors:
+    mirror: "s3://spack-binaries-develop/radiuss"
+
+  specs:
+  - matrix:
+    - [$radiuss]
+    - [$compilers]
+
+  gitlab-ci:
+    image: { "name": "ghcr.io/scottwittenburg/ecpe4s-ubuntu18.04-runner-x86_64:2020-09-01", "entrypoint": [""] }
+    script:
+      - . "./share/spack/setup-env.sh"
+      - spack --version
+      - cd share/spack/gitlab/cloud_pipelines/stacks/radiuss
+      - spack env activate --without-view .
+      - spack config add "config:install_tree:projections:${SPACK_JOB_SPEC_PKG_NAME}:'morepadding/{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'"
+      - spack -d ci rebuild
+    mappings:
+      - match: [sundials, umpire, vtk-h, vtk-m]
+        runner-attributes:
+          image: { "name": "ghcr.io/scottwittenburg/ecpe4s-ubuntu18.04-runner-x86_64:2020-09-01", "entrypoint": [""] }
+          tags: ["spack", "public", "xlarge", "x86_64"]
+      - match: ['os=ubuntu18.04']
+        runner-attributes:
+          image: { "name": "ghcr.io/scottwittenburg/ecpe4s-ubuntu18.04-runner-x86_64:2020-09-01", "entrypoint": [""] }
+          tags: ["spack", "public", "large", "x86_64"]
+    temporary-storage-url-prefix: "s3://spack-binaries-prs/pipeline-storage"
+    service-job-attributes:
+      image: { "name": "ghcr.io/scottwittenburg/ecpe4s-ubuntu18.04-runner-x86_64:2020-09-01", "entrypoint": [""] }
+      before_script:
+        - . "./share/spack/setup-env.sh"
+        - spack --version
+        - cd share/spack/gitlab/cloud_pipelines/stacks/radiuss
```

Comment:
#22887 does this here too, instead of using the concrete environment directory as I suggested in another comment.  I think this is fine since of the jobs that use `service-job-attributes`, only `rebuild-index` might actually need the environment activated, but likely only to know the mirror url, not for the concrete specs.  I admit it's a little unclear though.